### PR TITLE
Pause Play Functionality

### PR DESCRIPTION
**Pull request by billyendres**
- Add pause play feature for camera app
- Test on Chromebox Commercial
- Remove camera light on pause function

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-    <m-camera-barcode-scanner width="800" height="600"></m-camera-barcode-scanner>
+    <m-camera-barcode-scanner
+      width="800"
+      height="600"
+    ></m-camera-barcode-scanner>
     <script>
-        const reader = document.querySelector('m-camera-barcode-scanner');
-        reader.addEventListener('result', ({data}) => {
-            console.info(data);
+      let reader = document.querySelector("m-camera-barcode-scanner");
```
Comment (JacobWischnatAopen) [inline index.html]: Hey Billy, This let can stay as a const? Like it was previously?

**Review by JacobWischnatAopen — CHANGES_REQUESTED** [review]
IGNORE ME, I AM DUMB

**Review by JacobWischnatAopen — APPROVED** [review]

**Review by MileAit — CHANGES_REQUESTED** [review]
1) Please publish the latest version of `@meldcx/matrix-component-camera-barcode-scanner` to NPM (should be `1.0.6`)
2) Please bump the dependency version in `package.json` to `1.0.6`
3) Please switch back to the reference instead of copying the full implementation into example
4) Please switch back to the previous formatting (single quotes in scripts)
5) Please update readme
6) Please remove the `dist`, this is an example - not a production app

NPM URL to validate: 
https://www.npmjs.com/package/@meldcx/matrix-component-camera-barcode-scanner

**Review by MileAit — CHANGES_REQUESTED** [review]
Several more things left to do:
1. Please switch back to the previous formatting (single quotes in scripts)
2. Please add the new `pause()` example into the `index.html`
3. Please update the `readme.md` with the description on how to use the new `pause()` method

**Review by MileAit — APPROVED** [review]

**Pull request merged**